compiler/damlc/daml-ide-core/BUILD.bazel

```diff
@@ -6,9 +6,19 @@ load(
     "da_haskell_library",
 )
 
+filegroup(
+    name = "hlint-yaml",
+    srcs = [
+        "data/prod/hlint.yaml",
```

Comment:
Would be great to have a comment explaining the difference between the two files.

Reply:
Ok. The reason is this : at this time there are only a couple of hints available to us from the `ghc-lib-parser` code path in hlint : (1) module export hints and (2) newtype hints. (1) is set to ignore by default (by hlint) and (2) is enabled by default. We need tests that hlint is working so with the `test/hlint.yaml`, we let (2) reach our unit tests. With `prod/hlint.yaml` we disable (2) hints reaching the IDE. This is because the template desugaring code causes (2) to fire which is very annoying.

Reply:
This is the difference between the two files. The prod version has this:
```
- ignore: {name: "Use newtype instead of data"}
```